app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentEffectHandler.kt

```diff
@@ -40,6 +43,8 @@ class ScheduleAppointmentEffectHandler @AssistedInject constructor(
         .addTransformer(LoadDefaultAppointmentDate::class.java, loadDefaultAppointmentDate())
         .addConsumer(ShowDatePicker::class.java, { uiActions.showManualDateSelector(it.selectedDate) }, schedulers.ui())
         .addTransformer(LoadCurrentFacility::class.java, loadCurrentFacility())
+        .addTransformer(ScheduleAppointment::class.java, scheduleAppointment())
+        .addAction(CloseScreen::class.java, uiActions::closeSheet, schedulers.ui())
```

Comment:
I initially decided to keep it as `CloseScreen` because I felt that this being a "Sheet" is an implementation detail and not relevant to the Mobius loop. But I can rename this.

Reply:
Done.

Reply:
This aligns with the rest of the sheets though. Also the method names are still called `closeSheet`. Maybe we can move to calling it `finishUi` or something.

Reply:
Yeah, not really important at the moment, so maybe we can revisit this after moving completely to the navigation component and these screens are no longer activities.